tools/example-templates/additional-files/Attr3Attribute

```diff
@@ -0,0 +1,5 @@
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+public class Attr3Attribute : Attribute
```

Comment:
This looks exactly like Attr1Attribute - do we need both to demonstrate two attributes applied to the same class?

Reply:
And in fact, this doesn't have a file extension - is that deliberate?

Reply:
They are equivalent. Both are used in the example in which the declarations of the attributes types is irrelevant. I suggest leaving this as is.

See my first comment for this PR. I inadvertently created this file without an extension.